tests/components/deconz/test_sensor.py

```diff
@@ -275,6 +275,49 @@ async def test_no_sensors(
             "next_state": "50",
         },
     ),
+    (  # Carbon dioxide sensor
+        {
+            "capabilities": {
+                "measured_value": {
+                    "unit": "PPB",
```

Comment:
Why is this PPB but the sensor is in PPM?  I don't see any unit conversion code in sensor.py.

Reply:
Thanks, it was copy paste from other sensor, corrected based on your comment